modules/govuk/manifests/apps/rummager/rabbitmq.pp

```diff
@@ -52,4 +53,13 @@
     routing_key   => '*.links',
     create_queue  => false,
   }
+
+  # This queue should replace `rummager_to_be_indexed`
+  govuk_rabbitmq::queue_with_binding { 'govuk_index_durable':
```

Comment:
I was assuming it was the other way - you can define the pattern up front and it applies to anything create afterwards. I'll make sure to test this.